app/src/js/utils/table-config/pdrs.js

```diff
@@ -106,12 +117,16 @@ export const granuleTableColumns = [
   {
     Header: 'Name',
     accessor: 'granuleId',
+    isLink: true,
```

Comment:
General question: Was the table link handling changed to support the router update?

Reply:
Yes